src/pages/WorkPage/CommentSection/CommentInput/index.tsx

```diff
@@ -0,0 +1,100 @@
+import { useCallback, useRef, useState } from "react";
+
+import styles from "./index.module.css";
+import SendButton from "./SendButton";
+
+import type React from "react";
+import type { Comment } from "../../../../shared/types/comment";
+
+interface CommentInputProps {
+  onSubmit?: (message: string) => void;
+  replyingTo?: Comment | null;
+  onCancelReply?: () => void;
+  autoFocus?: boolean;
+}
```

Comment:
onSubmitはオプショナルじゃなくて良いかな
あとavatarUrlとかをオプショナルで持たせておくと良いかも
```suggestion
interface CommentInputProps {
  onSubmit: (message: string) => void;
  replyingTo?: Comment | null;
  onCancelReply?: () => void;
  autoFocus?: boolean;
  avatarUrl?: string;
}
```